### DISCUSSION TITLE
[Projects Beta] When Converting a Task to An Issue Assign Project

### DISCUSSION THREAD

**Discussion by mitchelsellers** [community/community#6294] [Projects and Issues]
When you create a new issue with sub-tasks within it.  You can assign the Issue to a particular project.

When you convert individual tasks to "issues" the project does not carry over, so then you have to manually go to each of these newly created issues and add them to the board.

Since the issue was tagged to a project, converted new issues I believe logically should be associated as well

**Comment by itcarroll** [comment]
I'd like to suggest a refinement to @mitchelsellers suggestion that would also add a feature to projects. There are two ways to view an issue that as been added to a (beta) project: from the repository or from the project. Currently, only the view from the repository has a button that converts items in a task list to a new issue. The view from the project should have that button too; and I'd suggest that only that button copy project data to the new issue.

In summary, when viewing an issue with a task list ...

... from a repository, any item in the task list has a button that converts it to a new issue but ignores all project attributes
... from a project, any item in the task list has a button that converts it to a new issue in the same project
